libmscore/chord.cpp

```diff
@@ -2285,11 +2285,9 @@ void Chord::layoutTablature()
                   }
             }
       if (!tab->genDurations()            // if tab is not set for duration symbols
-            || track2voice(track())       // or not in first voice
-            || isGrace()) {               // or chord is a grace (no dur. symbols for graces
```

Comment:
OK - I think this would be a good solution, that seems to work quite good:
      if (!tab->genDurations()                         // if tab is not set for duration symbols
            || track2voice(track())                    // or not in first voice
            || (isGrace()                              // no tab duration symbols if grace notes
                && beamMode() == Beam::Mode::AUTO)) {  // and beammode == AUTO